zooniverse/front-end-monorepo
# app-project: Add OrganizationLink to ProjectHeader

## Package
- app-project

## Linked Issue and/or Talk Post
- towards #3579 
- next step from #4567

## Describe your changes
- adds OrganizationLink component to ProjectHeader

## How to Review
_Helpful explanations that will make your reviewer happy:_
- What Zooniverse project should my reviewer use to review UX?
  - staging project with organization: https://local.zooniverse.org:3000/projects/markb-panoptes/uwworgtest-chicago-wildlife
  - production test project with translated organization title: https://local.zooniverse.org:3000/projects/fr/markbouslog/digileap-testing-production?env=production&demo=true
- What user actions should my reviewer step through to review this PR?
  - go to projects linked, note link to organization in project header
  - change language in production test project linked, note organization title change
  - go to project like I Fancy Cats without organization, note no changes
- Which storybook stories should be reviewed?
  - http://localhost:9001/?path=/story/project-app-shared-project-header--organization-link
- Are there plans for follow up PR’s to further fix this bug or develop this feature? yes, next PR will be for the organization link in Hero component

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General

## General UX
Example Staging Project: [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/assistive-technology/testing-steps/voiceover-mac.html)
  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - It is passing accessibility checks in the storybook

## New Feature
  - Example of SlideTutorial [component](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/components/Classifier/components/SlideTutorial/SlideTutorial.js) with docgen comments, and its [story](https://zooniverse.github.io/front-end-monorepo/@zooniverse/classifier/index.html?path=/docs/other-slidetutorial--default)

## Post-merge
